app/models/miq_task.rb

```diff
@@ -225,7 +225,7 @@ def state_or_status
   end
 
   def human_status
-    self.class.human_status(state_or_status)
+    _(self.class.human_status(state_or_status))
```

Comment:
@DavidResende0 This looks like it should go even deeper in the class method.  The class method is the one that accesses the constants, and those constants are defined with N_, so the class method seems like the best spot.